charts/argo-cd/README.md

```diff
@@ -34,6 +34,20 @@ Changes in the `CustomResourceDefinition` resources shall be fixed easily by cop
 
 ## Upgrading
 
+### 3.2.* 
+
+With this minor version we introduced the evaluation for the ingress manifest (depending on the capabilities version), See [Pull Request](https://github.com/argoproj/argo-helm/pull/637). [This Issues](https://github.com/argoproj/argo-helm/issues/703) reported that the capabilities evaluation is not handled correctly on clusters running a cluster version prior to `1.19` (which misses the `Ingress.networking.k8s.io`).
+
+If you are running a cluster version prior to `1.19` you can avoid this issue by directly installing chart version `3.3.0` and setting `global.kubeCapabilities` like:
+
+``` 
+global:
+  kubeCapabilities: "1.18.0"
+``` 
+
+Then you should no longer encounter this issue.
+
+
```

Comment:
We should highlight that this only happens when we deploy the chart with Argo. For all other cases where the `argo-cd` chart is deployed by other infrastructure provisioning tools (eg. Terraform, Flux, scripts), this can be ignored.